src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1201,6 +1221,13 @@ private static NewtonRaphsonStoppingCriteria createNewtonRaphsonStoppingCriteria
         }
     }
 
+    static List<AcOuterLoop> createOuterLoops(LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt) {
```

Comment:
So, just to be totally clear:
- If we have the parameter `outerLoopNames`, this override totally the parameters that defines the mode (phase control mode, transformer voltage control mode, shunt voltage control mode).
-  But it seems that it overrides also the booleans that defines if controls are enabled or not. Is it really what you want ? I am not sure we will change the list each time we want to disable a control (and I do it all the time).

Reply:
I agree with Anne.

Reply:
Fixed, now control status is also used in explicit outer loops configuration